db/migrate/20210108110945_install_audited.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class InstallAudited < ActiveRecord::Migration[6.1]
+  def self.up
+    create_table :audits, force: true do |t|
+      t.column :auditable_id, :integer
+      t.column :auditable_type, :string
+      t.column :associated_id, :integer
+      t.column :associated_type, :string
+      t.column :user_id, :integer
+      t.column :user_type, :string
+      t.column :username, :string
+      t.column :action, :string
+      t.column :audited_changes, :text
```

Comment:
I think there's an option to store the `audited_changes` as `jsonb` - it will be much easier to query the changes if we store as `jsonb` rather than text.

Reply:
Nice!